packages/web-app-files/tests/unit/helpers/collaborators.spec.js

```diff
@@ -0,0 +1,89 @@
+import {
+  permissionsBitmask,
+  roleToBitmask,
+  bitmaskToRole,
+  checkPermission
+} from '../../../src/helpers/collaborators.js'
+
+describe('collaborators', () => {
+  it('permissionBitmask', () => {
+    expect(permissionsBitmask).toMatchObject({
+      read: 1,
+      update: 2,
+      create: 4,
+      delete: 8,
+      share: 16,
+      all: 31
+    })
+  })
+
+  describe('roleToBitmask function', () => {
+    describe('if additionalPermissions', () => {
+      it('should return the sum of role permissions and additional permissions bitmasks if the given role has provided additional permission', () => {
```

Comment:
create more variations of the test, e.g. also test corner cases, invalid input, interesting combinations of the permissions etc.